### PR TITLE
Clarify allas boto3 instructions

### LINKED ISSUE #2142: Minor thing: Not to helpful section on Allas and boto3, missing explanation of credentials file
I found this section not very helpful:
https://github.com/CSCfi/csc-user-guide/blob/master/docs/data/Allas/using_allas/python_boto3.md#configuring-s3-credentials

The links leads to another page that does not explain credentials files. I have an Allas Access Key and a Secret Key, and neither the section nor the linked page helped me how to use those for accessing my buckets via boto3. The linked page tells me to install all kinds of packages to then run allas-conf, but all I needed was how to add the secret key/access key to the ~/.aws/credentials` file.

This one then helped, which I found googling:
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#shared-credentials-file

Maybe this could be made easier for future S3 noobs :) Thanks!

### PR DESCRIPTION
**Pull request by msainio**
## Proposed changes

Updated instructions on configuring credentials. Reworked content at start of page. Closes #2142.

Preview page:
https://csc-guide-preview.rahtiapp.fi/origin/clarify-allas-boto3-instructions/data/Allas/using_allas/python_boto3/

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes the automated tests. (If not, visit [pull requests at Travis CI](https://app.travis-ci.com/github/CSCfi/csc-user-guide/pull_requests) and have a look at the job log.)
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

File: docs/data/Allas/using_allas/python_boto3.md [hunk 1]
```diff
@@ -1,57 +1,102 @@
...
+supercomputer in `~/.aws/credentials`, where `boto3` looks for them
+automatically.
+
+If you are accessing Allas from a personal workstation,
+you can copy `.aws` and its contents to your workstation home directory
+[using a file transfer tool](../../moving/index.md), e.g. by running
+`scp -r <username>@puhti.csc.fi:~/.aws $HOME`. As above, `boto3` automatically
+looks for your credentials in your local home directory.
+
+Note that only one set of credentials can be stored in `~/.aws/credentials`.
```
Comment (ktiits) [inline docs/data/Allas/using_allas/python_boto3.md:69]: This is not true, see https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html and profile_name. I have several profiles in my credentials file from earlier. 
(On a conference today and then for 4 weeks on holidays, so can not properly review this.)

Comment (msainio) [reply docs/data/Allas/using_allas/python_boto3.md:69]: Thank you for pointing that out! I will make the appropriate changes.

Comment (ktiits) [reply docs/data/Allas/using_allas/python_boto3.md:69]: That might be true. You can ask also Robin or Henrik, I would recall that Puhti/Mahti/LUMi web interface Allas connection made credential per project, might remember wrong.

Comment (msainio) [reply docs/data/Allas/using_allas/python_boto3.md]: Thanks! That works :)

**Review by kkmattil — APPROVED** [review]

**Pull request merged**